### DISCUSSION TITLE
Zigbee How send tuya.sendDataPointBool(entity, keyid, value === 'ON')

### DISCUSSION THREAD

**Discussion by qlwz** [arendst/Tasmota#13134] [Q&A] [answered]
[https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/tuya.js#L823](https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/tuya.js#L823)

[https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/converters/toZigbee.js#L2844](https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/converters/toZigbee.js#L2844)

[https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/lib/tuya.js#L454](https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/lib/tuya.js#L454)

How to send this data

@s-hadinger

**Comment by s-hadinger** [comment, marked as answer]
Moving to discussions. Let me look if there's an easy way to do this.

**Reply by s-hadinger** [reply]
I'm struggling in understanding Z2M code since it has very deep ramifications. Do you know the format of the messages to send?

**Reply by s-hadinger** [reply]
Let's try this:
`ZbSend {"Device":"<device>","Write":{"EF00/0101":1}}`

For endpoints 1/2/3/4 try "EF00/0101" to "EF00/0104" and change value between 1 and 0 to send true or false.

Please use `WebLog 3` and paste logs here

**Reply by qlwz** [reply]
Thank you, this code can be sent
